csharp/src/Google.Protobuf.Test/JsonFormatterTest.cs

```diff
@@ -1,698 +1,705 @@
-        [TestCase(typeof(FloatValue), 1.5f, "1.5")]
-        [TestCase(typeof(DoubleValue), 1.5d, "1.5")]
-        public void Wrappers_Standalone(System.Type wrapperType, object value, string expectedJson)
-        {
-            IMessage populated = (IMessage)Activator.CreateInstance(wrapperType);
-            populated.Descriptor.Fields[WrappersReflection.WrapperValueFieldNumber].Accessor.SetValue(populated, value);
-            Assert.AreEqual(expectedJson, JsonFormatter.Default.Format(populated));
-        }
-
-        // Sanity tests for WriteValue. Not particularly comprehensive, as it's all covered above already,
-        // as FormatMessage uses WriteValue.
-
-        [TestCase(null, "null")]
-        [TestCase(1, "1")]
-        [TestCase(1L, "'1'")]
-        [TestCase(0.5f, "0.5")]
-        [TestCase(0.5d, "0.5")]
-        [TestCase("text", "'text'")]
-        [TestCase("x\ny", @"'x\ny'")]
-        [TestCase(ForeignEnum.ForeignBar, "'FOREIGN_BAR'")]
-        public void WriteValue_Constant(object value, string expectedJson)
-        {
-            AssertWriteValue(value, expectedJson);
-        }
-
-        [Test]
-        public void WriteValue_Timestamp()
-        {
-            var value = new DateTime(1673, 6, 19, 12, 34, 56, DateTimeKind.Utc).ToTimestamp();
-            AssertWriteValue(value, "'1673-06-19T12:34:56Z'");
-        }
-
-        [Test]
-        public void WriteValue_Message()
-        {
-            var value = new TestAllTypes { SingleInt32 = 100, SingleInt64 = 3210987654321L };
-            AssertWriteValue(value, "{ 'singleInt32': 100, 'singleInt64': '3210987654321' }");
-        }
-
-        [Test]
-        public void WriteValue_List()
-        {
-            var value = new RepeatedField<int> { 1, 2, 3 };
-            AssertWriteValue(value, "[ 1, 2, 3 ]");
-        }
-
-        [Test]
-        public void Proto2_DefaultValuesWritten()
-        {
-            var value = new ProtobufTestMessages.Proto2.TestAllTypesProto2() { FieldName13 = 0 };
-            AssertWriteValue(value, "{ 'FieldName13': 0 }");
-        }
-
-        private static void AssertWriteValue(object value, string expectedJson)
-        {
-            var writer = new StringWriter();
-            JsonFormatter.Default.WriteValue(writer, value);
-            string actual = writer.ToString();
-            AssertJson(expectedJson, actual);
-        }
-
-        /// <summary>
-        /// Checks that the actual JSON is the same as the expected JSON - but after replacing
-        /// all apostrophes in the expected JSON with double quotes. This basically makes the tests easier
-        /// to read.
-        /// </summary>
-        private static void AssertJson(string expectedJsonWithApostrophes, string actualJson)
-        {
-            var expectedJson = expectedJsonWithApostrophes.Replace("'", "\"");
-            Assert.AreEqual(expectedJson, actualJson);
-        }
-    }
-}
+﻿#region Copyright notice and license

```

Comment:
what has changes about this file that it shows up in the diff?
Unless I missed something, the entire file showing up as modified makes it impossible to see any actual changes to the file (which makes reviewing it very difficult).

Let's avoid "false diffs" like this in the future.

Reply:
I believe it's whitespace-only - quite possibly line endings. I'd really like to see the whitespace-only changes here reverted before we merge; it makes it much hard to trace changes.